science/nds2-client-swig/Portfile

```diff
@@ -48,6 +45,15 @@ configure.args    -DPYTHON=false \
                   -DCMAKE_BUILD_TYPE=RelWithDebInfo \
                   -DCMAKE_EXPORT_COMPILE_COMMANDS=1
 
+variant swig3 description {Use SWIG 3.x for generation of extensions} {
+    
+    depends_build-append    port:swig3
+    configure.args-append \
+        -DSWIG_EXECUTABLE:FILEPATH=${prefix}/bin/swig3
+}
```

Comment:
Why have you added a swig3 variant? What happens if the variant is deselected? I see several places in the port that reference swig even if the variant is not selected.

Reply:
I was setting up for the future for having a swig4 variant. Currently, swig(swig 4.x) does not work but steps are being taken to support it.

Reply:
If I try to install the port and disable the swig3 variant, it fails:
```
CMake Error at /opt/local/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find SWIG (missing: SWIG_EXECUTABLE SWIG_DIR) (Required is at
  least version "3.0.7")
```
So I would not make this a variant.

Even if it were compatible with swig 4, I don't see why the user would want to specify which swig to use. Just make the port depend on the most recent swig with which it is compatible.

Reply:
Variant has been eliminated